index 2.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="ru">
+ <head>
+   <meta charser="utf-8">
+   <title> Глэйси </title>
+ </head>
+
+
+ <body>
+   <header>
+   <nav>
+     <a>
+     <img src='Лого.png' alt='' width='154' height='64'>
+     </a>
+     <ul>
+       <li><a href="">Каталог </a></li>
+       <li><a href="">Доставка и оплата</a></li>
+       <li><a href="">О Компании </a></li>
+     </ul>
+     <a href="">Поиск</a>
+     <a href=""> Вход</a>
+     <a href="">Корзина</a>
+     <p>с 10 до 20, ежедневно</p>
+     <a href="tel:8 812 450-25-25">8 812 450-25-25</tel></a>
```

Comment:
```suggestion
     <a href="tel:88124502525">8 812 450-25-25</a>
```
Круто что сразу начал делать через `tel`, но нужно немного подправить